models/reviews.models.js

```diff
@@ -43,4 +43,20 @@ exports.fetchReviews = () => {
     return db.query(queryStr).then((response) => {
         return response.rows;
     });
+};
+
+exports.fetchReviewComments = (review_id) => {
+    const queryStr = `SELECT * FROM comments WHERE review_id = $1`
+
+    return db.query(queryStr, [review_id]).then((response) => {
+        if(!response.rows.length) {
+            return db.query(`SELECT * FROM reviews WHERE review_id = $1`, [review_id]).then((response) => {
```

Comment:
you're next .thens/promises here, let's stay away from this. See if you can refactor these 2 queries into a single promise using Promise.all